net/microsocks/files/microsocks.config

```diff
@@ -0,0 +1,7 @@
+config microsocks microsocks
+	option bindaddr 192.168.33.2
+ 	option listenip 10.41.0.4
```

Comment:
From microsocks's [README.md](https://github.com/rofl0r/microsocks/blob/master/Makefile), the default listeinip is 0.0.0.0.

Reply:
I would avoid listening on all interfaces by default. Looks like part of recipe to get open proxy by defaults.
127.0.0.1 is better default IMHO